specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```
@@ -1643,6 +1643,423 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/datasets": {
+      "get": {
+        "tags": [
+          "MachineLearningDatasets"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "MachineLearningDataset_ListByWorkspace",
```

Please change this operationId to align with the others
```suggestion
        "operationId": "MachineLearningDatasets_ListByWorkspace",
```